skyreader/result.py

```diff
@@ -301,7 +304,7 @@ def is_close(
         Returns:
             bool: True if `other` and `self` are close
         """
-        if not rtol_per_field:
+        if rtol_per_field is None:
```

Comment:
This is a fine change, but it'd be good to have a new unit test for when this value is `{}`.